docs/0-what-is/1-getting-started.mdx

```diff
@@ -5,81 +5,191 @@ sidebar_position: 0
-configurations.
+## Our goal
 
-## Example pipeline
+In this guide, our goal will be to stream information about flights from an
+imaginary airport to a file. The flight records contain just the airline name
+and the scheduled departure time.
 
-:::tip
+## Install Conduit
 
-In our [Conduit repository](https://github.com/ConduitIO/conduit), you can find [more examples](https://github.com/ConduitIO/conduit/tree/main/examples/pipelines), but to ilustrate a simple use case we'll show a pipeline using a file as a source, and another file as a destination. Check out the different [specifications](/docs/using/pipelines/configuration-file) to see the different configuration options.
+If you're using a macOS or Linux system, you can install Conduit with the
+following command:
 
+```shell
+$ curl https://conduit.io/install.sh | bash
+```
+
+If you're not using macOS or Linux system, you can still install Conduit
+following one of the different options provided
+in [our installation page](/docs/installing-and-running).
+
+:::note
+The Conduit binary contains both, the Conduit service and the Conduit CLI, with
+which you can interact with Conduit.
 :::
 
-Create a folder called `pipelines` at the same level as your Conduit binary.
-Inside of that folder create a file named `file-to-file.yml`.
+## Initialize Conduit 
+
+First, let's create a directory where we will build our first pipeline:
+
+```shell
+$ mkdir ~/conduit-playground/ && cd ~/conduit-playground/
+```
+
+Now, let's initialize the "working environment" in this directory:
+
+```shell
+$ conduit init
+
+Created directory: processors
+Created directory: connectors
+Created directory: pipelines
+Configuration file written to conduit.yaml
+
+Conduit has been initialized!
+
+To quickly create an example pipeline, run 'conduit pipelines init'.
+To see how you can customize your first pipeline, run 'conduit pipelines init --help'.
+```
+
+`conduit init` creates the directories where you can put your pipeline
+configuration files, connector binaries, and processor binaries. There's also a
+`conduit.yaml` that contains all the configuration parameters Conduit supports.
+
+In this guide, we'll only use the `pipelines` directory, since we won't need to
+install any additional connector nor to change Conduit's default configuration.
+
+## Build a pipeline
+
+Next, we can use the Conduit CLI to build the example pipeline:
 
 ```shell
-│ # Conduit binary
-├── conduit
-│ # Folder with pipeline configurations
-└── pipelines
-    │ # Pipeline configuration file
-    └── file-to-file.yml
+$ conduit pipelines init
 ```
 
-Copy following content into `file-to-file.yml`:
+`conduit pipelines init` builds an example that generates flight information
+from an imaginary airport every second. Use `conduit pipelines init --help` to
+learn how to customize the pipeline.
+
+If the `pipelines` directory, you'll notice a new file,
+`pipeline-generator-to-file.yaml` that contains our pipeline's configuration:
```

Comment:
I would add a call out as 

```
:::tip 

Here's a full detailed information about Pipeline configuration files (/docs/using/pipelines/configuration-file)

:::
```

Reply:
We have that under "What's next?" or do you think we should move it here?

Reply:
@hariso you're right. That's enough.